src/app/Client/SubmitWriting.tsx

```diff
@@ -0,0 +1,67 @@
+'use client';
+
+import { useCallback, useContext } from "react";
+import { PromptContext } from "./Contexts/PromptFeatures";
+import { Resolver, SubmitHandler, useForm } from "react-hook-form";
+import * as yup from "yup";
+import { yupResolver } from "@hookform/resolvers/yup";
+
+type Inputs = {
+  story: string
+}
+
+const SubmitWriting = () => {
+  const {
+    requiredWord,
+    genre,
+    action,
+    name,
+    line,
+  } = useContext(PromptContext);
+
+  const schema = yup.object().shape({
+    story: yup
+      .string()
+      .required("You must enter a story")
+      .test("contains-required-word", `The story must contain the word "${requiredWord}"`, function(value) {
+        return value?.includes(requiredWord);
+      }),
+  });
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<Inputs>({ resolver: yupResolver(schema) });
+
+  const onSubmit: SubmitHandler<Inputs> = (data) => {
+    console.log(data.story);
```

Comment:
ok for now.  will need to actually save to a database ideally; requires knowledge of what. database options are available on vercel presumably or whatever free hosting i go with.